kinder/pkg/cluster/manager/actions/setup-external-ca.go

```diff
@@ -72,6 +72,18 @@ func SetupExternalCA(c *status.Cluster, vLevel int) error {
 		return nil
 	}
 
+	generateKubeletConfWorker := func(n *status.Node) error {
+		if err := n.Command(
+			"/bin/sh", "-c",
+			fmt.Sprintf("kubeadm init phase kubeconfig kubelet --control-plane-endpoint=%s --apiserver-advertise-address=%s --v=%d",
```

Comment:
i guess that's one way of working around without post processing with sed or something else. nice.

Reply:
Yeah, first wanted to do sed but noticed this works too and does not require an additional command :-)